BatterySensor.h

```diff
@@ -55,6 +55,7 @@ class NoBattery {
 
 
 class InternalVCC {
+  uint16_t vcc;
```

Comment:
Brauchen wir die ?

Reply:
Ähm, nein.
Die hatte ich kurz eingeführt um in `start()` zu messen und in `finish()` zurückzugeben.